src/htmlhighlighter.test.js

```diff
@@ -10,11 +10,25 @@ import * as attest from '../test/attest';
 
 const { assert } = chai;
 
+let hl;
+let is;
+
 // Test specifications
 describe('HTML Highlighter', function() {
-  let hl;
-  let is;
+  describeGeneralTests();
+  describeCursorMovementTests();
+  describeTextSelectionTests();
+  describeXpathTests();
+  describeSpecialCharacterHandlingTests();
+
+  try {
+    describeFullDocumentTests();
+  } catch (x) {
+    console.warn('Full document tests NOT available in browser mode');
```

Comment:
Should we just log the `x` here? Looks like the message is the same.

Reply:
Doing so would pollute the output generated by the test runner and would add zero benefit since the only exception produced is the one inside `describeFullDocumentTests`; `describeFullDocumentTests` only parses and exposes the tests defined within and is not expected to ever produce additional exceptions. No logging necessary therefore.